peers/app_request_netrwork_metrics.go

```diff
@@ -0,0 +1,54 @@
+package peers
+
+import (
+	"errors"
+	"github.com/ava-labs/awm-relayer/config"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	ErrFailedToCreateAppRequestNetworkMetrics = errors.New("failed to create app request network metrics")
+)
+
+type AppRequestNetworkMetrics struct {
+	infoAPIBaseURL   string
+	pChainAPIBaseURL string
+
+	infoAPICallLatencyMS   *prometheus.HistogramVec
+	pChainAPICallLatencyMS *prometheus.HistogramVec
+}
+
+func NewAppRequestNetworkMetrics(cfg *config.Config, registerer prometheus.Registerer) (*AppRequestNetworkMetrics, error) {
+	infoAPICallLatencyMS := prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "info_api_call_latency_ms",
+			Help:    "Latency of calling info api in milliseconds",
+			Buckets: prometheus.LinearBuckets(10, 10, 10),
```

Comment:
Rather than linearly spaced buckets, exponentially distributed buckets would be more useful for identifying latency spikes. I think a good distribution to use would be to measure between ~0.1s and ~10s latency (a reasonable timeout), with bucket doubling the previous bucket's range. This can be done with [ExponentialBucketsRange](https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#ExponentialBuckets).